### PR TITLE
Leverage more historical data in Finances window

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Inspired by Locomotion and RCT Classic, this PR introduces a horizontal scrollbar to the Finances window's summary page. Up to 14 months of historical data are kept in the SV6 format, which this PR leverages.

The scroll area is initialised all the way to the right, so casual players shouldn't be confused by this new enhancement.

### Screenshot
evergreen gardens 2018-03-12 18-34-50

As a bonus, a screenshot of the same save's summary tab [before the change](https://user-images.githubusercontent.com/604665/37299656-3473dbe8-2624-11e8-9216-49b631b696b8.png).

### CODE REVIEW

**Comment by devnoname120** [comment]
This change is really great!

I have a suggestion, though. Would it be possible to automatically scroll to the right when the slider is on the rightmost and a new month is added? With this pull request, you currently need to manually scroll to the right every time a new month appears.

**Comment by AaronVanGeffen** [comment]
> With this pull request, you currently need to manually scroll to the right every time a new month appears.

That is indeed the case for games with less than 15 months of financial history.

> Would it be possible to automatically scroll to the right when the slider is on the rightmost and a new month is added?

Yes, that should be doable. I'll have to take care not to reset the scroll position when it's been manually adjusted, though.

**Comment by devnoname120** [comment]
> I'll have to take care not to reset the scroll position when it's been manually adjusted

True, but if it has been manually adjusted to the right-most, I think that it should automatically scroll to the right too.

**Comment by AaronVanGeffen** [comment]
That's a fair point. I'll take it into account, thanks.

**Comment by AaronVanGeffen** [comment]
Using any of the scrollbar widgets does not appear to fire a specific event — only interacting with the scrolling area does. I've therefore opted to always reset the scrolling position at the start of a new month.

File: src/openrct2-ui/windows/Finances.cpp [hunk 1]
```diff
@@ -517,7 +525,7 @@ rct_window * window_finances_open()
 
     w = window_bring_to_front_by_class(WC_FINANCES);
     if (w == nullptr) {
-        w = window_create_auto_pos(530, 290, _windowFinancesPageEvents[0], WC_FINANCES, WF_10);
+        w = window_create_auto_pos(530, 310, _windowFinancesPageEvents[0], WC_FINANCES, WF_10);
```
Comment (Broxzier) [inline src/openrct2-ui/windows/Finances.cpp]: Would be nice to have these in defines, so there's only one place to change them when needed.

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/Finances.cpp]: Like a `WW` and `WH`? Trouble is, each of the tabs have different widths and heights, as you may recall. Properly refactoring that is beyond the scope of this PR, I think.

**Pull request merged**